openfoodfacts/types.py

```diff
@@ -129,6 +129,31 @@ def from_str_or_enum(cls, value: Union[str, "Facet"]) -> "Facet":
                 raise ValueError("unknown Facet: %s", value)
             return cls[value]
 
+class SearchFilterOperator(str, enum.Enum):
```

Comment:
Would it make sense for this to inherit from `enum.StrEnum` instead of `str, enum.Enum`? Though I guess `str, enum.Enum` is how other classes here are doing it, so probably just a thought for future, rather than something you should change for this PR. :)